### PR TITLE
Add Publishing section

### LINKED ISSUE #275: Get Publishing page to appear in the index
**Problem:**
In PR #219, a new page was created: [publishing.md](https://gtfs.org/schedule/publishing/).
This page does not appear in the site Index. 

**Solution proposed:**
Add the page in the index on the left and as a button like the other sections of the site.

### PR DESCRIPTION
**Pull request by isabelle-dr**
Attempt to close #275. In this PR:
- Adding a "Publishing" section in the Index and as a button on the GTFS Schedule Landing page.

### CODE REVIEW

File: docs/schedule/index.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 # GTFS Schedule Overview
 
 <div class="landing-page">
-    <a class="button" href="reference">Reference</a><a class="button" href="best-practices">Best Practices</a><a class="button" href="examples">Examples</a><a class="button" href="changes">Changes</a><a class="button" href="validate">Validate</a>
+    <a class="button" href="reference">Reference</a><a class="button" href="best-practices">Best Practices</a><a class="button" href="examples">Examples</a><a class="button" href="changes">Changes</a><a class="button" href="validate">Validate</a><a class="button" href="publish">Publish</a>
```
Comment (tzujenchanmbd) [inline docs/schedule/index.md:8]: Probably related to this discussion? 
https://mobilitydata-io.slack.com/archives/C06026D89G8/p1704812528356999

Adding a tab in the index lgtm.

**Review by tzujenchanmbd — APPROVED** [review]
lgtm!

File: docs/schedule/index.md (hunk 1 above)
Comment (isabelle-dr) [reply docs/schedule/index.md:8]: Yes.
After discussion, we're keeping this Publishing Resource as part of GTFS Schedule for now.

**Comment by isabelle-dr** [comment]
Thank you @tzujenchanmbd, merging this PR.

**Pull request merged**